mlrun/common/model_monitoring/stream_processing.py

```diff
@@ -21,12 +21,7 @@
 import pandas as pd
 import storey
 
-import mlrun
-import mlrun.common.model_monitoring
-import mlrun.config
-import mlrun.datastore.targets
 import mlrun.feature_store.steps
-import mlrun.utils
 import mlrun.utils.model_monitoring
```

Comment:
I'll remove `import mlrun.utils.model_monitoring` as well to avoid that external monitoring code import. We imported it for `parse_model_endpoint_store_prefix()` method so I'll move this method to `mlrun.common.model_monitoring.helpers.py`.